libs/methodologies/bold/recycling/organic/types/src/document.types.ts

```
@@ -1,43 +1,16 @@
-import type { UnknownObject } from '@carrot-fndn/shared/types';
+import type { MethodologyDocument as MethodologyDocument } from '@carrot-fndn/shared/types';
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)